provisioning/RHCSA_RHCE_LAB/tasks/main.yml

```diff
@@ -11,33 +11,28 @@
     - libselinux-python
     - python-pip # for tzupdate
     - bash-completion
-    - bash-completion-extras
-    - dbus # for ansible nmcli http://docs.ansible.com/ansible/nmcli_module.html
```

Comment:
Ansible mcli doesn't work properly [unless initiated from another CentOS/RH client](https://github.com/ansible/ansible-modules-extras/issues/1130)
> At the time of writing the plugin, I hadn't targeted OSX or Ubuntu as a platform that this would be running.
Alternatively we could run the [Ansible_local provider](https://www.vagrantup.com/docs/provisioning/ansible_local.html)